Training/PythonWebServer/resources/stylesheet.css

```diff
@@ -1,5 +1,136 @@
-h1 {
+p{
+	display: inline;
+}
+html{
+	height: 100%;
+}
+body{
+	height: 100%;
+	margin: 0;
+	background-repeat: no-repeat;
+	background: linear-gradient(grey, white);
+	background-attachment: fixed;
+}
+.wrapper{
+	display: block;
+	margin-right: auto;
+	margin-left: auto;
+	width: 80%;
+	text-align: center;
+}
+.button{
+	box-shadow: 7px 7px grey;
+	border-radius: 5px;
+
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 4cm;
+	width: 330px;
+	height: 70px;
+
+	font-family: sans-serif;
+	text-align: center;
+	color: white;
+	font-weight: bold;
+	
+	background-color: #F5D04C;
+    border: none;
+    padding: 10px 75px;
+    font-size: 40px;
+    display: block;
```

Comment:
CSS spacing was a nightmare before, but then came flexbox! I'd highly recommend trying to use flexbox to get all your spacing right.

https://css-tricks.com/snippets/css/a-guide-to-flexbox/